src/core/helpers/texts.ts

```diff
@@ -2,3 +2,10 @@ export const beautifyText = (str: string) => {
   if (!str) return str;
   return (str.charAt(0).toUpperCase() + str.slice(1)).replaceAll('_', ' ');
 };
+
+export const areEqual = (str1: string, str2: string) => {
```

Comment:
It's not a general function. If it is only used for handling the 'description' and 'name' attributes in the Schema form, please move it to its own service